kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/SchemaManagerTest.java

```diff
@@ -47,6 +47,7 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
```

Comment:
@mubeen can you please revert all changes to this file as none appear necessary?

Reply:
Hi @C0urante 

File was reverted back to commit 26ecf5e9f8d3d524cea1dec999d10a222f74539f